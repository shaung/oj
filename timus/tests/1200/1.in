-1.00 34.86
200
