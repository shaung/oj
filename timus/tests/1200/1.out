303.62
0 17